app/controllers/articles_controller.rb

```diff
@@ -1,11 +1,12 @@
 class ArticlesController < BaseController
```

Comment:
protip: you can place this one liner under the class declaration in order to run a certain method before all controller actions

```
before_action :bind_skills_and_disciplines
```